### PR TITLE
Skeleton doc for technical resources

### PR DESCRIPTION
**Pull request by hx-markterry**
This is the layout i'm using for technical resources that new starters can use as a base to get themselves up to speed.

Ideally each header will contain links but this will take some time to gather this information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ This is a collection of documents that outline Holiday Extras' culture and devel
  * [Coding Principles](coding-principles.md)
  * [Organising code in backbone projects](organising-code-backbone-projects.md)
  * [PR template](pr-template.md)
- * [Holiday Extras Security Policy](security-policy.md)
```
Comment (hpoom) [inline README.md:14]: Did you intend to remove this?

**Comment by hpoom** [comment]
:+1:  From me @hx-markterry 

Once this is merged give me a shout. I would love to do some PR's to add links into many of those sections.

File: technical-resources.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Below is a list of tech and concepts that we expect our developers to know either before they start or be trained on after starting with us:
+
+We do not expect our developers to be experts in all the areas below, just to have a one or more clues about how these items work.
+
+The links below should provide a good introduction into an item, further in depth links can also be added.
+
+## Clientside basics
+
+### JQuery
```
Comment (dancork) [inline technical-resources.md]: J should be lowercase.

File: technical-resources.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+### Git
+
+* [Git For Humans](https://speakerdeck.com/dancork/2015)
+
+### Shell (bash)
+
+### SQL
+
+### CSS
```
Comment (dancork) [inline technical-resources.md]: Should this go under clientside basics?

File: README.md (hunk 1 above)
Comment (hx-markterry) [inline README.md:14]: Yup, it should of been removed a while ago when we removed this page.

**Pull request merged**